NewForce-Cohort-9/newforce-c9-tabloidmvc-TabloidMVC
# Index View for Tag Management Complete

## Description
Added logged in admin Index view for Tag Management link in navbar. This lists all tags in the database alphabetically, as requested in [ticket #13] (https://trello.com/c/rRHKX46x/13-13-view-all-tags)

## Testing Instructions
- git add and git commit whatever you're working on on your branch
- git checkout main
- git fetch --all
- git checkout <rev-tag-index>
- If you don't see the most recent version of my files, git pull origin <rev-tag-index>
- If you haven't created the Tabloid MVC database, please do so.
- Click the execute with debugger solid green button to launch the app
- In browser, please go to Login on the nav bar and log in with this email: admin@example.com
- This should redirect you back to the home page.
- Click on the link in the Navbar titled "TAG MANAGEMENT"
- This should take you to the /tag URL and display an alphabetized list of all tags from the database. 
- Log out should be showing on the nav bar now. Please click this to log out.
- You should see a friendly message letting you know you are logged out of the application.

### Checklist:
- [] My code follows the style guidelines of this project
- [] I have performed a self-review of my code
- [] I have commented my code, particularly in hard-to-understand areas
- []  My changes generate no new warnings or errors
- []  I have added test instructions that prove my fix is effective or that my feature works
